RELEASING

```diff
@@ -0,0 +1,25 @@
+# Scala.js Release Process
+
+1. Clean up bugs:
+    - Make sure all bugs assigned to the milestone are closed.
+    - Make sure all fixed bugs are assigned to the milestone ([this query][1]
+      should not return anything).
+1. Create a "Version x.y.z." commit ([example][2]) and push it to a branch on your fork.
+1. Ping people on the commit for review.
+1. Testing (post results as comments to commit):
+    - Nightly
+    - Weekly
+    - [Manual testing][3]
+1. If all tests pass and you have LGTM, push to master and tag.
+1. Perform [manual testing][3] that needs the tagging (source maps).
+1. Publish:
+    - Sonatype, bintray
```

Comment:
This is basically running `./script/publish.sh` (with `-x` for the non-dry run).
So far I am the only one with the credentials to do so, however.